src/energy.cpp

```diff
@@ -291,24 +291,31 @@ double PolicyIonIon::surfaceEnergy(const EwaldData &d, Change &change, Space::Tg
 }
 
 double PolicyIonIon::selfEnergy(const EwaldData &d, Change &change, Space::Tgvec &groups) {
```

Comment:
`PolicyIonIon::selfEnergy()` is never used as the self-energy is injected by the pair-potential which in this case means from the external CoulombGalore.